analyzer/decrypt_crypto.cc

```diff
@@ -314,14 +322,45 @@ std::vector<uint8_t> HP_INFO = {0x00, 0x10, 0x0d, 0x74, 0x6c, 0x73, 0x31, 0x33,
 
 class QuicPacketProtectionV1 : public QuicPacketProtection {
 public:
-    virtual std::vector<uint8_t>& GetInitialSalt() override { return INITIAL_SALT_V1; }
+    virtual bool Supports(uint32_t version) const override {
+        // Quic V1
+        if ( version == 0x00000001 )
+            return true;
+
+        // Draft 22 through 34
+        if ( version >= 0xff000016 && version <= 0xff000022 )
+            return true;
+
+        // mvfst from facebook
+        if ( version == 0xfaceb001 || version >= 0xfaceb002 && version <= 0xfaceb013 )
```

Comment:
Nit: no need to let the user wonder about precedence of `||` and `&&`.

```suggestion
        if ( version == 0xfaceb001 || (version >= 0xfaceb002 && version <= 0xfaceb013) )
```